### ISSUE TITLE
Overlay glasses rendering issues [Plethora V1.1.12] [MC 1.12.2]

### ISSUE THREAD

**Issue by Shiranuit** [SquidDev-CC/ccjam-2016#145]
Hello, i have found some issues with the plethora overlay glasses
- [X] Plethora 1.1.12
- [x] Minecraft 1.12.2
- [X] Forge 14.23.4.2730
- [X] Windows 7

### addTriangle()
- When a triangle is added to the canvas, his color is wrong and change with the block that i am looking, or if i have an item in the hand or not.

Code used : 
```lua
local p=peripheral.wrap("back")
local c=p.canvas()
c.addTriangle({0,0},{100,0},{0,100},0xFFFFFFFF)
```
Screenshots addTriangle :
![image](https://user-images.githubusercontent.com/19224148/43599225-3abf3758-9687-11e8-8e80-6bd38c853cb6.png)
![image](https://user-images.githubusercontent.com/19224148/43599242-4d64bf86-9687-11e8-8a28-304c80a51be8.png)
![image](https://user-images.githubusercontent.com/19224148/43599298-72f79480-9687-11e8-868b-afb9ebf14e3b.png)

### addLines
- When i add the thickness parameter the color of the lines change
Code without the thickness parameter:
```lua
local p=peripheral.wrap("back")
local c=p.canvas()
c.addLines({0,0},{100,0},{0,100},0xFFFFFFFF)
```
Screenshot without the thickness parameter:
![image](https://user-images.githubusercontent.com/19224148/43599968-904b3dc8-9689-11e8-98d6-efa6951e4334.png)

Code with the thickness parameter:
```lua
local p=peripheral.wrap("back")
local c=p.canvas()
c.addLines({0,0},{100,0},{0,100},0xFFFFFFFF,10)
```
Screenshot with the thickness parameter:
![image](https://user-images.githubusercontent.com/19224148/43599953-7fd73fe6-9689-11e8-9a1d-e6aa875ef596.png)

### addPolygon
- When created the Polygon got the wrong color
Code used:
```lua
local p=peripheral.wrap("back")
local c=p.canvas()
c.addPolygon({0,0},{100,0},{0,100},0xFFFFFFFF)
```

Screenshot:
![image](https://user-images.githubusercontent.com/19224148/43600267-72654f00-968a-11e8-96f2-30ce4a622032.png)

### ~~addItem~~
~~- When you add an item to the canvas in Singleplayer it work fine, but not in Multiplayer~~
~~Code used:~~
~~local p=peripheral.wrap("back")~~
~~local c=p.canvas()~~
~~c.addItem({0,0},"minecraft:stone")~~

~~Error:~~
~~Java Exception Thrown:~~
~~java.lang.NoClassDefFoundError: net/minecraft/client/entity/EntityPlayerSP~~
~~Screenshot:~~
![image](https://user-images.githubusercontent.com/19224148/43600599-594bc7fa-968b-11e8-82fa-7a705064dbf5.png)

**Comment by dmarcuse** [comment]
The `addItem` issue should have been fixed after #110. Try [1.1.12](https://github.com/SquidDev-CC/plethora/releases/tag/v1.12-1.1.12)?

**Comment by Shiranuit** [comment]
Correct, the addItem issue have been fixed in plethora 1.1.12

I checked too and the others issues in Plethora 1.1.12 are not resolved

**Comment by SquidDev** [comment]
I believe they should all be fixed now. I recently redid some of the rendering code, which should fix have fixed triangles being slightly odd (can no longer reproduce it any way). Other two should be fixed by the above. 

Thank you very much for the incredibly well put together big report. I cannot overstate how much it is appreciated!
